### ISSUE TITLE
Short pages seem to be blank when switched to from long page.

### ISSUE THREAD

**Issue by martinhbramwell** [awatson1978/starrynight#74]
If one is reading the bottom of the page "Testing" and then clicks a link to go to any other page, they all appear to be blank.  The workaround is to scroll back up, but it is too easy to jump to the more obvious conclusion that the site has failed completely.

**Comment by awatson1978** [comment]
Yeah, I think I know what you're talking about.  We need to set a $.scrollTo(0) when navigating to the next page.  Um, not going to be able to get to it right away; but putting this on the todo for the next sprint.  Thx for the bug report.
